### ISSUE TITLE
Asset issuer selection

### ISSUE THREAD

**Issue by CryptoKong** [Codaone/DEXBot#406]
Rather than having users type a prefix in the GUI, perhaps having a drop down menu next to the asset where the user can select cryptobridge, openledger, easydex, spark, etc. Then auto attach the prefix for them (in the background). If BTS or any smartcoin or non prefixed asset is typed in, it shouldn't matter which exchange they select.

**Comment by MarkoPaasila** [comment]
This would guide the user to input correct values, and thus reduce silly errors and frustration. Good suggestion. I see no downside.

"Native" should be one option. Think BTC.

**Comment by CryptoKong** [comment]
Yes good idea. The native option would probably need explaining in the "?" as some may be unsure as to its meaning, ie: a lot of cryptobridge users have no idea about bitshares.

**Comment by thehapax** [comment]
@MarkoPaasila  Would it be ok if i look into this? I need something to do to stay current and i haven't looked into the GUI yet. Plus I have questions about how joelva approaches development here, and something small would be good to look at. Totally understand if no budget but I also need to be prepared for wp2 on the front end.

**Comment by thehapax** [comment]
ok. @MarkoPaasila
